xrpl4j-integration-tests/src/test/java/org/xrpl/xrpl4j/tests/BinarySerializationTests.java

```diff
@@ -386,7 +388,7 @@ void serializeTrustLine() throws JsonProcessingException {
       .build();
 
 
-    String expectedBinary = "1200142200020000240000002C63D6438D7EA4C68000000000000000000000000000574347000000" +
+    String expectedBinary = "1200142280020000240000002C63D6438D7EA4C68000000000000000000000000000574347000000" +
```

Comment:
I assume this change due to default canonical setting?

Reply:
Exactly. The only difference is in the 9th character